zulip_bots/zulip_bots/test_lib.py

```diff
@@ -159,6 +159,13 @@ def read_bot_fixture_data(bot_name, test_name):
     http_data = json.loads(content)
     return http_data
 
+def get_response(http_response, http_headers):
+    # type: (Dict[str, Any], Dict[str, Any]) -> Any
+    mock_result = requests.Response()
+    mock_result._content = json.dumps(http_response).encode()  # type: ignore # This modifies a "hidden" attribute.
+    mock_result.status_code = http_headers.get('status', 200)
+    return mock_result
```

Comment:
Since this method is only used in `mock_http_conversation`, it probably makes sense to define it directly in `mock_http_conversation`.

Reply:
That way, we also avoid the trouble with finding a decent name for this function that is not  too confusing for the general namespace.